src/test/java/edu/wright/airviewer2/SplitAndMergeTest.java

```diff
@@ -0,0 +1,123 @@
+import org.apache.pdfbox.multipdf.Splitter;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.junit.Test;
+import org.testfx.framework.junit.ApplicationTest;
+
+import javafx.application.Platform;
+import javafx.scene.Group;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Pagination;
+import javafx.scene.control.Alert.AlertType;
+import javafx.scene.image.ImageView;
+import javafx.stage.DirectoryChooser;
+import javafx.stage.FileChooser;
+
+
+
+public class SplitAndMergeTest extends ApplicationTest {
+	
+	
+	@Test
+	public void mergefileTest()  {
+		try {
+		Pagination pagination=  new Pagination();
+		Path currentDir = Paths.get(".");
+		String current = currentDir.toAbsolutePath() + "/src/test/resources/test.pdf";
+		Path exist = Paths.get(current);
+		AIRViewerModel model = new AIRViewerModel(exist);
+		pagination.setPageCount(model.numPages());
+	    pagination.setDisable(true);
+	    Group pageImageGroup = new Group();
+	    pagination.setPageFactory(index -> {
+	    	System.out.println("haiinpagena");
+            ImageView currentPageImageView = new ImageView(model.getImage(0));
+            pageImageGroup.getChildren().clear();
+            pageImageGroup.getChildren().add(currentPageImageView);
+   
+			return pageImageGroup;
+        });
+        File f = new File(current);
+		//when(fileChooser.showOpenDialog(any()).thenReturn(fileChooser);
+		Platform.runLater(new Runnable(){
+
+			@Override
+			public void run() {
+				// TODO Auto-generated method stub
+				try {
+					new SplitAndMerge().mergefile(pagination,model);
+				} catch (Exception e) {
+					// TODO Auto-generated catch block
+					e.printStackTrace();
+				}
+				
+				
+			}
+			// do your GUI stuff here
+			});
+		} catch(Exception e) {
+			e.printStackTrace();
+		}
+	}
+	
+	@Test
+	public void splitterTest() {
+		try {
+			Pagination pagination=  new Pagination();
+			Path currentDir = Paths.get(".");
+			String current = currentDir.toAbsolutePath() + "/src/test/resources/test.pdf";
+			Path exist = Paths.get(current);
+			AIRViewerModel model = new AIRViewerModel(exist);
+			pagination.setPageCount(model.numPages());
+		    pagination.setDisable(true);
+		    Group pageImageGroup = new Group();
+		    pagination.setPageFactory(index -> {
+		    	System.out.println("haiinpagena");
+	            ImageView currentPageImageView = new ImageView(model.getImage(0));
+	            pageImageGroup.getChildren().clear();
+	            pageImageGroup.getChildren().add(currentPageImageView);
+	   
+				return pageImageGroup;
+	        });
+	        File f = new File(current);
+			//when(fileChooser.showOpenDialog(any()).thenReturn(fileChooser);
+			Platform.runLater(new Runnable(){
+
+				@Override
+				public void run() {
+					// TODO Auto-generated method stub
+					try {
+						new SplitAndMerge().splitter("1",pagination,model);
+					} catch (Exception e) {
+						// TODO Auto-generated catch block
+						e.printStackTrace();
+					}
+					
```

Comment:
remove the more no of empty lines and format the code

Reply:
Removed the empty lines and formatted code. Please review it